main.go

```diff
@@ -24,14 +25,16 @@ func parseConfig() (config, error) {
 	result := config{
 		ListenAddr: ":9205",
 		Timeout:    5 * time.Second,
+		Username:   getEnv("NEXTCLOUD_USERNAME", ""),
```

Comment:
As the current default is `""` for all values this can probably be simplified to just use `os.Getenv()`.